src/studentView.pug

```diff
@@ -3,17 +3,49 @@ include NodeSearch.pug
 include TabSwitcher.pug
 
 #studentView(hidden='')
-    include header.pug
-    +node-search
-    //-
-        .tab-switcher
-            ul
-                li.tab
-                    button Upcoming Assignments
-                li.tab
-                    button Practice Equations
-                li.tab
-                    button View Grades
+    header
```

Comment:
This looks wrong. It seems like things are being put back in after Jon's changes... can you confirm this?

Reply:
Did Jon add that?